unicodetools/src/main/java/org/unicode/text/UCA/WriteCollationData.java

```diff
@@ -1859,21 +1857,12 @@ public static UCA getCollator(CollatorType type) {
         switch (type) {
             case cldr:
                 if (cldrCollator == null) {
-                    //                if (Default.ucdVersion().compareTo("6.1") < 0) { // only
-                    // reorder if less than v6.1
-                    //                    cldrCollator = buildCldrCollator(true);
-                    //                } else
-                    {
-                        cldrCollator = buildCldrCollator(false);
-
-                        cldrCollator.overrideCE("\uFFFE", 0x1, 0x20, 2);
-                        cldrCollator.overrideCE("\uFFFF", 0xFFFE, 0x20, 2);
```

Comment:
FYI: These overrides are both here and inside buildCldrCollator(boolean). The ones inside are used by passing in true. Seems cleaner inside. (See also issue #794)

Reply:
Minor: better to use an a meaningful enum rather than a boolean, so that people can tell immediately what 
`cldrCollator = buildCldrCollator(true);` means rather than guess ("does false mean 'don't build'??"). Better to have `cldrCollator = buildCldrCollator(UCA.Style.addFFFx);`


Not a blocker though!